packages/app-bridge/src/AppBridgeTheme.ts

```diff
@@ -252,7 +259,7 @@ export interface AppBridgeTheme<
 
     updateBrandportalLink(brandportalLink: Partial<BrandportalLink>): Promise<Partial<BrandportalLink> | null>;
 
-    getCoverPage(): Promise<CoverPage>;
+    getCoverPage(language?: string): Promise<CoverPage>;
```

Comment:
Does the caller of this function need to have control over the `language` parameter?

Reply:
Removed